src/main/java/com/plugin/frontend/actions/FormatAreaAction.java

```diff
@@ -15,60 +16,70 @@
 
 import java.io.*;
 import java.util.ArrayList;
-import java.util.Objects;
 
 public class FormatAreaAction extends AnAction {
+    private int startOffset;
```

Comment:
You use only one public method from the class. It is not permissible to declare variables here.

Reply:
So where I should declare variables????

Reply:
Discussed